src/Module/Command.php

```diff
@@ -18,7 +18,7 @@ class Command
     /**
      * @var string
      */
-    private $projectDir = '.';
+    public $projectDir = '.';
```

Comment:
To make testing possible. See [this line in ExpressiveCommand](https://github.com/zendframework/zend-expressive-tooling/pull/22/files#diff-b240d71305c9faddf69a01e76b75fdabR158); without that, we cannot do test cases to ensure delegation works correctly for commands that utilize the filesystem.